jquery.panelSnap.js

```diff
@@ -535,10 +600,15 @@ if ( typeof Object.create !== 'function' ) {
     delay: 0,
     easing: 'linear',
     offset: 0,
-    keyboardNavigation: {
-      enabled: false,
-      nextPanelKey: 40,
-      previousPanelKey: 38,
+    navigation: {
+      keys: {
+        nextKey: 40,
+        prevKey: 38
```

Comment:
These should be false or null by default. Otherwise keyboard navigation will be enabled by default.